Building a project fails - missing module Data.Aeson.Parser

Building a project with the Servant dependency fails with `Could not find module ‘Data.Aeson.Parser`.

I'm using GHC 9.6.2 and Cabal 3.10.1.0 (both managed by GHCup) on a Macbook M1 Pro.

Partial log:
```
[ 4 of 34] Compiling Servant.API.ContentTypes ( src/Servant/API/ContentTypes.hs, dist/build/Servant/API/ContentTypes.o, dist/build/Servant/API/ContentTypes.dyn_o )

src/Servant/API/ContentTypes.hs:79:1: error:
    Could not find module ‘Data.Aeson.Parser’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
79 | import           Data.Aeson.Parser
```

Complete log:
```
Configuring library for servant-0.20..
Preprocessing library for servant-0.20..
Building library for servant-0.20..
[ 1 of 34] Compiling Servant.API.Alternative ( src/Servant/API/Alternative.hs, dist/build/Servant/API/Alternative.o, dist/build/Servant/API/Alternative.dyn_o )

src/Servant/API/Alternative.hs:10:1: warning: [-Wunused-imports]
    The import of ‘Control.Applicative’ is redundant
      except perhaps to import instances from ‘Control.Applicative’
    To import instances alone, use: import Control.Applicative()
   |
10 | import           Control.Applicative (liftA2)
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Servant/API/Alternative.hs:41:5: warning: [-Wnoncanonical-monoid-instances]
    Noncanonical ‘mappend’ definition detected
    in the instance declaration for ‘Monoid (a :<|> b)’.
    ‘mappend’ will eventually be removed in favour of ‘(<>)’
    Either remove definition for ‘mappend’ (recommended) or define as ‘mappend = (<>)’
    See also: https://gitlab.haskell.org/ghc/ghc/-/wikis/proposal/semigroup-monoid
   |
41 |     (a :<|> b) `mappend` (a' :<|> b') = (a `mappend` a') :<|> (b `mappend` b')
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ 2 of 34] Compiling Servant.API.BasicAuth ( src/Servant/API/BasicAuth.hs, dist/build/Servant/API/BasicAuth.o, dist/build/Servant/API/BasicAuth.dyn_o )

src/Servant/API/BasicAuth.hs:27:47: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
27 | data BasicAuth (realm :: Symbol) (userData :: *)
   |                                               ^
[ 3 of 34] Compiling Servant.API.Capture ( src/Servant/API/Capture.hs, dist/build/Servant/API/Capture.o, dist/build/Servant/API/Capture.dyn_o )

src/Servant/API/Capture.hs:20:25: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
20 | data Capture' (mods :: [*]) (sym :: Symbol) (a :: *)
   |                         ^

src/Servant/API/Capture.hs:20:51: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
20 | data Capture' (mods :: [*]) (sym :: Symbol) (a :: *)
   |                                                   ^

src/Servant/API/Capture.hs:30:39: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
30 | data CaptureAll (sym :: Symbol) (a :: *)
   |                                       ^
[ 4 of 34] Compiling Servant.API.ContentTypes ( src/Servant/API/ContentTypes.hs, dist/build/Servant/API/ContentTypes.o, dist/build/Servant/API/ContentTypes.dyn_o )

src/Servant/API/ContentTypes.hs:79:1: error:
    Could not find module ‘Data.Aeson.Parser’
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
79 | import           Data.Aeson.Parser
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...
[ 5 of 34] Compiling Servant.API.Description ( src/Servant/API/Description.hs, dist/build/Servant/API/Description.o, dist/build/Servant/API/Description.dyn_o )

src/Servant/API/Description.hs:62:57: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
62 | type family FoldDescription' (acc :: Symbol) (mods ::  [*]) :: Symbol where
   |                                                         ^
[ 6 of 34] Compiling Servant.API.Empty ( src/Servant/API/Empty.hs, dist/build/Servant/API/Empty.o, dist/build/Servant/API/Empty.dyn_o )
[ 7 of 34] Compiling Servant.API.Experimental.Auth ( src/Servant/API/Experimental/Auth.hs, dist/build/Servant/API/Experimental/Auth.o, dist/build/Servant/API/Experimental/Auth.dyn_o )
[ 8 of 34] Compiling Servant.API.Fragment ( src/Servant/API/Fragment.hs, dist/build/Servant/API/Fragment.o, dist/build/Servant/API/Fragment.dyn_o )

src/Servant/API/Fragment.hs:17:21: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
17 | data Fragment (a :: *)
   |                     ^
[ 9 of 34] Compiling Servant.API.Generic ( src/Servant/API/Generic.hs, dist/build/Servant/API/Generic.o, dist/build/Servant/API/Generic.dyn_o )

src/Servant/API/Generic.hs:86:25: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
86 |     type mode :- api :: *
   |                         ^
[10 of 34] Compiling Servant.API.HttpVersion ( src/Servant/API/HttpVersion.hs, dist/build/Servant/API/HttpVersion.o, dist/build/Servant/API/HttpVersion.dyn_o )
[11 of 34] Compiling Servant.API.IsSecure ( src/Servant/API/IsSecure.hs, dist/build/Servant/API/IsSecure.o, dist/build/Servant/API/IsSecure.dyn_o )
[12 of 34] Compiling Servant.API.Modifiers ( src/Servant/API/Modifiers.hs, dist/build/Servant/API/Modifiers.o, dist/build/Servant/API/Modifiers.dyn_o )

src/Servant/API/Modifiers.hs:54:51: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
54 | type family FoldRequired' (acc :: Bool) (mods :: [*]) :: Bool where
   |                                                   ^

src/Servant/API/Modifiers.hs:75:51: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
75 | type family FoldLenient' (acc :: Bool) (mods ::  [*]) :: Bool where
   |                                                   ^
[13 of 34] Compiling Servant.API.Header ( src/Servant/API/Header.hs, dist/build/Servant/API/Header.o, dist/build/Servant/API/Header.dyn_o )

src/Servant/API/Header.hs:26:24: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
26 | data Header' (mods :: [*]) (sym :: Symbol) (a :: *)
   |                        ^

src/Servant/API/Header.hs:26:50: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
26 | data Header' (mods :: [*]) (sym :: Symbol) (a :: *)
   |                                                  ^
[14 of 34] Compiling Servant.API.NamedRoutes ( src/Servant/API/NamedRoutes.hs, dist/build/Servant/API/NamedRoutes.o, dist/build/Servant/API/NamedRoutes.dyn_o )

src/Servant/API/NamedRoutes.hs:10:26: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
10 | data NamedRoutes (api :: * -> *)
   |                          ^

src/Servant/API/NamedRoutes.hs:10:31: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
10 | data NamedRoutes (api :: * -> *)
   |                               ^
[15 of 34] Compiling Servant.API.QueryParam ( src/Servant/API/QueryParam.hs, dist/build/Servant/API/QueryParam.o, dist/build/Servant/API/QueryParam.dyn_o )

src/Servant/API/QueryParam.hs:24:28: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
24 | data QueryParam' (mods :: [*]) (sym :: Symbol) (a :: *)
   |                            ^

src/Servant/API/QueryParam.hs:24:54: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
24 | data QueryParam' (mods :: [*]) (sym :: Symbol) (a :: *)
   |                                                      ^

src/Servant/API/QueryParam.hs:38:40: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
38 | data QueryParams (sym :: Symbol) (a :: *)
   |                                        ^
[16 of 34] Compiling Servant.API.Raw  ( src/Servant/API/Raw.hs, dist/build/Servant/API/Raw.o, dist/build/Servant/API/Raw.dyn_o )
[17 of 34] Compiling Servant.API.RemoteHost ( src/Servant/API/RemoteHost.hs, dist/build/Servant/API/RemoteHost.o, dist/build/Servant/API/RemoteHost.dyn_o )
[18 of 34] Compiling Servant.API.ReqBody ( src/Servant/API/ReqBody.hs, dist/build/Servant/API/ReqBody.o, dist/build/Servant/API/ReqBody.dyn_o )

src/Servant/API/ReqBody.hs:24:25: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
24 | data ReqBody' (mods :: [*]) (contentTypes :: [*]) (a :: *)
   |                         ^

src/Servant/API/ReqBody.hs:24:47: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
24 | data ReqBody' (mods :: [*]) (contentTypes :: [*]) (a :: *)
   |                                               ^

src/Servant/API/ReqBody.hs:24:57: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
24 | data ReqBody' (mods :: [*]) (contentTypes :: [*]) (a :: *)
   |                                                         ^
[19 of 34] Compiling Servant.API.Status ( src/Servant/API/Status.hs, dist/build/Servant/API/Status.o, dist/build/Servant/API/Status.dyn_o )
[20 of 34] Compiling Servant.API.Sub  ( src/Servant/API/Sub.hs, dist/build/Servant/API/Sub.o, dist/build/Servant/API/Sub.dyn_o )

src/Servant/API/Sub.hs:18:27: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
18 | data (path :: k) :> (a :: *)
   |                           ^
[21 of 34] Compiling Servant.API.TypeErrors ( src/Servant/API/TypeErrors.hs, dist/build/Servant/API/TypeErrors.o, dist/build/Servant/API/TypeErrors.dyn_o )
[22 of 34] Compiling Servant.API.UVerb.Union ( src/Servant/API/UVerb/Union.hs, dist/build/Servant/API/UVerb/Union.o, dist/build/Servant/API/UVerb/Union.dyn_o )

src/Servant/API/UVerb/Union.hs:78:16: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
78 |   forall (c :: * -> Constraint) (a :: *) (as :: [*]).
   |                ^

src/Servant/API/UVerb/Union.hs:78:39: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
78 |   forall (c :: * -> Constraint) (a :: *) (as :: [*]).
   |                                       ^

src/Servant/API/UVerb/Union.hs:78:50: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
78 |   forall (c :: * -> Constraint) (a :: *) (as :: [*]).
   |                                                  ^

src/Servant/API/UVerb/Union.hs:91:28: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
91 | matchUnion :: forall (a :: *) (as :: [*]). (IsMember a as) => Union as -> Maybe a
   |                            ^

src/Servant/API/UVerb/Union.hs:91:39: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
91 | matchUnion :: forall (a :: *) (as :: [*]). (IsMember a as) => Union as -> Maybe a
   |                                       ^
[23 of 34] Compiling Servant.API.ResponseHeaders ( src/Servant/API/ResponseHeaders.hs, dist/build/Servant/API/ResponseHeaders.o, dist/build/Servant/API/ResponseHeaders.dyn_o )

src/Servant/API/ResponseHeaders.hs:39:1: warning: [-Wunused-imports]
    The import of ‘init, unlines’
    from module ‘Data.ByteString.Char8’ is redundant
   |
39 | import           Data.ByteString.Char8     as BS
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...

src/Servant/API/ResponseHeaders.hs:94:32: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
94 | type family HeaderValMap (f :: * -> *) (xs :: [*]) where
   |                                ^

src/Servant/API/ResponseHeaders.hs:94:37: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
94 | type family HeaderValMap (f :: * -> *) (xs :: [*]) where
   |                                     ^

src/Servant/API/ResponseHeaders.hs:94:48: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
94 | type family HeaderValMap (f :: * -> *) (xs :: [*]) where
   |                                                ^
[25 of 34] Compiling Servant.API.Vault ( src/Servant/API/Vault.hs, dist/build/Servant/API/Vault.o, dist/build/Servant/API/Vault.dyn_o )
[26 of 34] Compiling Servant.API.Verbs ( src/Servant/API/Verbs.hs, dist/build/Servant/API/Verbs.o, dist/build/Servant/API/Verbs.dyn_o )

src/Servant/API/Verbs.hs:29:64: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
29 | data Verb (method :: k1) (statusCode :: Nat) (contentTypes :: [*]) (a :: *)
   |                                                                ^

src/Servant/API/Verbs.hs:29:74: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
29 | data Verb (method :: k1) (statusCode :: Nat) (contentTypes :: [*]) (a :: *)
   |                                                                          ^
[28 of 34] Compiling Servant.API.WithNamedContext ( src/Servant/API/WithNamedContext.hs, dist/build/Servant/API/WithNamedContext.o, dist/build/Servant/API/WithNamedContext.dyn_o )

src/Servant/API/WithNamedContext.hs:21:56: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
21 | data WithNamedContext (name :: Symbol) (subContext :: [*]) subApi
   |                                                        ^
[29 of 34] Compiling Servant.API.WithResource ( src/Servant/API/WithResource.hs, dist/build/Servant/API/WithResource.o, dist/build/Servant/API/WithResource.dyn_o )
[30 of 34] Compiling Servant.Types.SourceT ( src/Servant/Types/SourceT.hs, dist/build/Servant/Types/SourceT.o, dist/build/Servant/Types/SourceT.dyn_o )

src/Servant/Types/SourceT.hs:73:19: warning: [GHC-58520] [-Wtype-equality-requires-operators]
    The use of ‘~’ without TypeOperators
    will become an error in a future GHC release.
    Suggested fix: Perhaps you intended to use TypeOperators
   |
73 | instance Identity ~ m => Foldable (SourceT m) where
   |                   ^

src/Servant/Types/SourceT.hs:126:19: warning: [GHC-58520] [-Wtype-equality-requires-operators]
    The use of ‘~’ without TypeOperators
    will become an error in a future GHC release.
    Suggested fix: Perhaps you intended to use TypeOperators
    |
126 | instance Identity ~ m => Foldable (StepT m) where
    |                   ^
[31 of 34] Compiling Servant.API.Stream ( src/Servant/API/Stream.hs, dist/build/Servant/API/Stream.o, dist/build/Servant/API/Stream.dyn_o )

src/Servant/API/Stream.hs:66:56: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
66 | data Stream (method :: k1) (status :: Nat) (framing :: *) (contentType :: *) (a :: *)
   |                                                        ^

src/Servant/API/Stream.hs:66:75: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
66 | data Stream (method :: k1) (status :: Nat) (framing :: *) (contentType :: *) (a :: *)
   |                                                                           ^

src/Servant/API/Stream.hs:66:84: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
66 | data Stream (method :: k1) (status :: Nat) (framing :: *) (contentType :: *) (a :: *)
   |                                                                                    ^

src/Servant/API/Stream.hs:75:28: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
75 | data StreamBody' (mods :: [*]) (framing :: *) (contentType :: *) (a :: *)
   |                            ^

src/Servant/API/Stream.hs:75:44: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
75 | data StreamBody' (mods :: [*]) (framing :: *) (contentType :: *) (a :: *)
   |                                            ^

src/Servant/API/Stream.hs:75:63: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
75 | data StreamBody' (mods :: [*]) (framing :: *) (contentType :: *) (a :: *)
   |                                                               ^

src/Servant/API/Stream.hs:75:72: warning: [GHC-39567] [-Wstar-is-type]
    Using ‘*’ (or its Unicode variant) to mean ‘Data.Kind.Type’
    relies on the StarIsType extension, which will become
    deprecated in the future.
    Suggested fix: Use ‘Type’ from ‘Data.Kind’ instead.
   |
75 | data StreamBody' (mods :: [*]) (framing :: *) (contentType :: *) (a :: *)
   |                                                                        ^
```

Relevant `cabal` file entries:
```
cabal-version:   3.0
build-type:      Simple

common warnings
  ghc-options: -Wall

build-depends:
  , aeson
  , base            ^>=4.18.0.0
  , bytestring
  , cryptonite
  , directory
  , filepath
  , Glob
  , mime-types
  , servant
  , servant-server
  , text
  , wai-cors
  , warp

default-language: Haskell2010
```

--- a/servant-server/servant-server.cabal
+++ b/servant-server/servant-server.cabal
@@ -73,7 +73,7 @@ library
   -- strict dependency as we re-export 'servant' things.
   build-depends:
       servant             >= 0.20     && < 0.21
-    , http-api-data       >= 0.4.1    && < 0.6
+    , http-api-data       >= 0.4.1    && < 0.7
 
   -- Other dependencies: Lower bound around what is in the latest Stackage LTS.
   -- Here can be exceptions if we really need features from the newer versions.

--- a/servant/servant.cabal
+++ b/servant/servant.cabal
@@ -93,14 +93,14 @@ library
   -- We depend (heavily) on the API of these packages:
   -- i.e. re-export, or allow using without direct dependency
   build-depends:
-      http-api-data          >= 0.4.1    && < 0.6
+      http-api-data          >= 0.4.1    && < 0.7
     , singleton-bool         >= 0.1.4    && < 0.2
 
   -- Other dependencies: Lower bound around what is in the latest Stackage LTS.
   -- Here can be exceptions if we really need features from the newer versions.
   build-depends:
       base-compat            >= 0.10.5   && < 0.14
-    , aeson                  >= 1.4.1.0  && < 3
+    , aeson                  >= 1.4.1.0  && < 2.2
     , attoparsec             >= 0.13.2.2 && < 0.15
     , bifunctors             >= 5.5.3    && < 5.7
     , case-insensitive       >= 1.2.0.11 && < 1.3

